### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by watchmexiang** [yaq007/Autofocus-Layer#8]
Hello, may I ask if this Autofocus Layer can directly replace Dilated convolution? How is the effect, thank you

**Comment by yaq007** [comment]
Hi, our experiments are designed to compare autofocus layer and dilated convolution and the results show that autofocus layer is better than dilated convolution.

**Comment by watchmexiang** [comment]
Thank you for your reply, may I ask if I can replace the ordinary convolution layer with it, will there be any improvement?

**Comment by yaq007** [comment]
Yes, this is what we claimed in our paper. A standard convolutional layer or dilated convolutional layer can be replaced by our autofocus layer to achieve better performance.
